.github/workflows/sync-rest.yml

```diff
@@ -0,0 +1,272 @@
+name: Sync Rest
+
+on:
+  workflow_dispatch:
+    inputs:
+      tracks:
+        description: 'List of tracks to rerun the syncing workflow for, formatted as JSON array, e.g. ["exercism/julia"]'
```

Comment:
This applies to the rest of this file: there are quite some mentions of tracks, but this workflow is aimed _not_ to deal with tracks, right?

Reply:
I've not renamed most mentions of tracks where they're used as a variable. That should make it easier to apply changes to all workflows and also makes it easier to reason about them. Ideally these should just be called repos but that'd have blown up the diff even more.